packages/cloudapi-gql/src/index.js

```diff
@@ -1,12 +1,29 @@
 const express = require('express');
+const graphqlHTTP = require('express-graphql');
 const cors = require('cors');
 
+const schema = require('./schema');
+
 const app = express();
 
 app.use(cors());
 app.options('*', cors());
 
-app.use('/graphql', require('./endpoint'));
+app.post(
+  '/graphql',
```

Comment:
We going to use the same endpint ?

Reply:
Ideally not, but GraphiQL doesn't play nice when we use different endpoints.

Reply:
Then cool, valid reason !